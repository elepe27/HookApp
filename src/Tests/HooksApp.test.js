
import { shallow } from 'enzyme'
import { HooksApp } from '../HooksApp'


describe('Pruebas en nuestro <HookApp/>', () => {

    test('Se debe ver el componente', () => {
        
        
        const wrapper = shallow(<HooksApp/>)
        expect(wrapper).toMatchSnapshot();



    })
    
    
})
