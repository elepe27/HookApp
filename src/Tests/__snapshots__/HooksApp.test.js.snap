// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en nuestro <HookApp/> Se debe ver el componente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
